README.md

```diff
@@ -10,10 +10,12 @@
 
 ## Setup and configuration
 
-Running this project is as simple as deploying it to a balenaCloud application. You can do it in just one click by using the button below:
+Running this app is as simple as deploying it to a balenaCloud application. You can do it in just one click by using the button below:
```

Comment:
Running this app is as simple as deploying it to a balenaCloud ~~application~~ fleet

Reply:
good catch